cfgov/flags/migrations/0001_initial.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file should be removed and replaced with [wagtail-flags](https://github.com/cfpb/wagtail-flags).